.github/workflows/build_test.yml

```diff
@@ -48,13 +48,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.8, 3.9, "3.10", "3.11" ]
+        python-version: [ 3.9, "3.10", "3.11" ]
     steps:
     - uses: actions/checkout@v3
     - name: Setup Python ${{ matrix.python-version }}
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Install DisCoPy 1.0.1 from GitHub
+      if: github.ref_name != 'release' && github.ref_name != 'beta'
+      run: pip install https://api.github.com/repos/discopy/discopy/zipball/pull/148/head
```

Comment:
This needs to be resolved before we can proceed with new lambeq release.

Reply:
This will be resolved with the new DisCoPy release coming out this week.